go.mod

```diff
@@ -1,137 +1,26 @@
-module github.com/kubernetes-sigs/aws-fsx-openzfs-csi-driver
```

Comment:
Will revert to the original module name. We should eventually switch over to sigs.k8s.io although.

Reply:
We actually made a switch from k8s.io to github in a recent PR, following precedent from EBS and Lustre. I don't think it actually matters because sigs.k8s.io redirects to github regardless. I'm assuming (blindly) that using github.com is better practice.

Reply:
Hm, I figured sigs.k8s.io was best practice. Looking at the major sigs packages, seems like they all use sigs.k8s.io.